### PR TITLE
Localize abstract-document to Vietnamese

### PR DESCRIPTION
**Pull request by m-inh**
I translate `abstract-document` pattern to Vietnamese

### CODE REVIEW

File: localization/vi/abstract-document/README.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
+---
+layout: pattern
+title: Abstract Document
+folder: abstract-document
+permalink: /patterns/abstract-document/
+categories: Structural
+language: vi
+tags: 
+ - Extensibility
+---
```
Comment (iluwatar) [inline localization/vi/abstract-document/README.md:10]: Please check the YAML frontmatter requirements in https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Comment by iluwatar** [comment]
Issue #2288

File: localization/vi/abstract-document/README.md (hunk 1 above)
Comment (m-inh) [reply localization/vi/abstract-document/README.md]: I don't know if I understand your requirement right. But I did remove `layout`, `folder`, `permalink`, and change `categories` to `category` already

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Thanks @m-inh, looks good!

**Pull request merged**